### DISCUSSION TITLE
想问一下能不能设置公招券始终保留一张

### DISCUSSION THREAD

**Discussion by CCCClaus** [MaaAssistantArknights/MaaAssistantArknights#2505] [Ideas]
如题，我想问问能不能设置公招券始终保留一张。提出这个需求是由于我注意到公招会强制刷新，并且maa一般情况下都会把公招券花完，如果在花完后刷出高资tag就没有办法了。

**Comment by horror-proton** [comment]
好像之前也有人提出过类似的想法, 不过我还没完全理解这个是什么情况, 按照设计如果公招资源用完了maa应该会停止公招任务并视为失败, 为什么会又去刷出来高资tag

**Comment by et2012** [comment]
公招券不够用大概率是花在招募3星干员上了，既然都用到一张不剩了，不妨考虑一下手动确认3星，只自动招募4星。
急缺绿票的时候用加急+公招券就行了（加急券价值几乎为零，不太可能用完的，留着公招券就等于绿票）。
另外，也可以限制MAA公招的执行次数。
